packages/components/src/CodeEditor.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from "react";
+import AceEditor from "react-ace";
+
+import "ace-builds/src-noconflict/mode-golang";
+import "ace-builds/src-noconflict/theme-github";
+import "ace-builds/src-noconflict/ext-language_tools";
+import "ace-builds/src-noconflict/keybinding-vim";
```

Comment:
Do we need all of these now? Maybe comment out the vim, when we're not using it?

Reply:
Oh, good catch! I'll fix this

Reply:
Fixed!